spec/spec.yaml

```diff
@@ -1432,6 +1432,73 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+    patch:
+      operationId: modifyClusterApp
+      tags:
+        - apps
+      summary: Modify app
+      description: |
+        This operation allows you to modify an existing app.
+
+        The following attributes can be modified:
+
+        - `user_config`: This lets you change which configmap is used to configure this app, as well as remove it entirely, by submitting null values.
```

Comment:
Removing the config is a good use case but then the OPA code might add it again? I think removing the values from the user CM is a better solution.

Reply:
So also when creating an app, there is no real need to make the user_config something that users can set themselves via the API right?

Reply:
I would like to support removing, and I do not think OPA should add user configs, OPA should only fill in cluster config

Reply:
Ack. That makes sense to me.